code/src/Albums.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Artist } from './Artists'
+import { HoverIcons } from 'Hovericons'
+
+
+// This compontent will fetch all data related to the Album including Album name and album image.
+export const Album = (props) => {
+  return (
+    <article className='album-card'>
+      <div className="Img-wrapper">
+        <HoverIcons />
+        <img src={props.item.images[1].url} alt='album cover' />
```

Comment:
Suggestion: having the album title as `alt`

Reply:
Great idea!